.github/workflows/run_pytest.yml

```diff
@@ -6,7 +6,7 @@ on:
 jobs:
   run_tests:
     env:
-      SECRET_KEY: ${{ secrets.SECRET_KEY }}
+      FLASK_APP_SECRET_KEY: ${{ secrets.SECRET_KEY }}
```

Comment:
does this instance need to change to `FLASK_APP_COOKIE_ENCRYPTION_KEY`? What about other instances of `SECRET_KEY` and `FLASK_APP_SECRET_KEY` in the code?

Reply:
Yes it does, thank you for catching that! Other instances have already been changed!